src/readJson.ts

```diff
@@ -1,5 +1,21 @@
 import { readFileSync } from 'fs';
+
+interface ConfigJson {
+  enforcedPackages?: string[];
+  ignoredBranchPrefixes?: string[];
+  launchdarklyClientId?: string;
+}
+
 export function readJson(filename: string) {
   return JSON.parse(readFileSync(filename).toString('utf-8'));
 }
 
+export function getConfig(configPath: string): ConfigJson {
+  try {
+    return readJson(configPath);
```

Comment:
instead of try-catch, let's throw the error

Reply:
I chose to swallow the error here because I try not to error installs on exceptions unrelated to peer deps. e.g. if products change their folder structure that causes the config not found error, we probably don't want that to block installs?